src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -0,0 +1,188 @@
+@import (reference) "../less/utilities";
+
+// @import (reference) "~clever-components/dist/less/index";
+
+@ICON_SMALL_SIZE: @size_2xl;
+@ICON_REGULAR_SIZE: @size_2xl;
+@HOVER_BACKGROUND_COLOR: #f9faff;
+@ERROR_RED: #df3537;
+@ERROR_RED_TEXT: #b30c10;
+@WHITE: #fff;
+
+.MessagingAttachment--ParentContainer {
+  position: relative;
+}
+
+.MessagingAttachment--Container {
+  z-index: auto;
+  width: 21.875rem; // 350px
+  cursor: pointer;
+  padding-left: 0.6875rem; // 11px
+  padding-right: 0.6875rem; // 11px
+  padding-top: 0.9375rem; // 15px
+  padding-bottom: 0.9375rem; // 15px
+  height: @ICON_REGULAR_SIZE;
+  border: @size_4xs solid @neutral_silver;
+  border-radius: @size_2xs;
+}
+
+.MessagingAttachment--IconContainer {
+  height: @ICON_REGULAR_SIZE;
+  .margin--right--s();
+  align-items: center;
+  justify-content: center;
+}
+
+// handle FontAwesome as well, for loading spinner
+.MessagingAttachment--IconContainer img,
+.MessagingAttachment--IconContainer .fa,
+.MessagingAttachment--IconContainer > * {
+  max-height: @ICON_REGULAR_SIZE;
+  max-width: @ICON_REGULAR_SIZE;
+  width: auto;
+  height: auto;
+}
+
+.MessagingAttachment--TextContainer {
+  justify-content: center;
+  color: @neutral_dark_gray;
+  max-width: 18.375rem; // 294px
+}
+
+// TODO: remove if unused
+.MessagingAttachment--Title {
+  font-size: @size_m;
+  line-height: @size_l;
+  .text--semi-bold();
+  .text--truncate();
+}
+
+.MessagingAttachment--Subtitle {
+  font-size: 0.875rem; // 14px
+  line-height: @size_m;
+  .text--truncate();
+}
+
+.MessagingAttachment--CloseButton {
+  z-index: 10;
+  position: absolute;
+  .flexbox();
+  top: -0.625rem; // 10px
+  right: -0.625rem; // 10px
+  width: 1.25rem; // 20px
+  height: 1.25rem; // 20px
+  border-radius: 50%;
+  cursor: pointer;
+  background-color: @neutral_medium_gray;
+  color: @WHITE;
+  border: none;
+  padding: 0;
+  border: @size_3xs solid white;
+
+  // TODO: doesn't turn off after click...
+  // maybe this doesn't matter cause clicking will close the box?
```

Comment:
Does this mean that the state persists after the click? Hm, since we expect the click to always _do something_, I think this might actually be fine, as you noted - the focus moves elsewhere anyway. 🤔

Reply:
I think it's because it's focussed after clicking. I could remove the styling for :focus, but I think that's worse for accessibility